src/Akeneo/Pim/Enrichment/Bundle/Controller/ExternalApi/ProductController.php

```diff
@@ -386,6 +388,15 @@ public function createAction(Request $request): Response
         $this->denyAccessUnlessAclIsGranted('pim_api_product_edit');
 
         $data = $this->getDecodedContent($request->getContent());
+        $violations = $this->validator->validate($data, new PayloadFormat());
+        if (0 < $violations->count()) {
+            $firstViolation = $violations->get(0);
+            throw new DocumentedHttpException(
+                Documentation::URL . 'patch_products__code_',
```

Comment:
Shouldn't this be :
```suggestion
                Documentation::URL . 'post_products',
```
?

Reply:
Good catch, it's fixed